tests/unittests/GraphOptzTest.cpp

```diff
@@ -273,16 +273,35 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvFP16) {
 
   EXPECT_EQ(F_->getNodes().size(), 3);
 
+  optimizedF_ = F_->clone(F_->getName().str() + "_optimized");
   ::glow::convertPlaceholdersToConstants(F_, bindings_, {});
-  ::glow::optimize(F_, CompilationMode::Infer);
-  EXPECT_EQ(F_->getNodes().size(), 2);
+  ::glow::convertPlaceholdersToConstants(optimizedF_, bindings_, {});
+  ::glow::optimize(optimizedF_, CompilationMode::Infer);
 
-  ASSERT_EQ(A->getNumUsers(), 1);
-  Node *newCV = A->getUsers().begin()->getUser();
+  EXPECT_EQ(optimizedF_->getNodes().size(), 2);
+
+  ASSERT_EQ(A->getNumUsers(), 2);
+  auto usersIt = A->getUsers().begin();
+  Node *oldCV = usersIt->getUser();
+  EXPECT_TRUE(llvm::isa<ConvolutionNode>(oldCV));
+  ASSERT_EQ(oldCV->getNumUsers(), 1);
+  Node *oldBN = oldCV->getUsers().begin()->getUser();
+  EXPECT_TRUE(llvm::isa<BatchNormalizationNode>(oldBN));
+  ASSERT_EQ(oldBN->getNumUsers(), 1);
+  Node *oldSave = oldBN->getUsers().begin()->getUser();
+  EXPECT_TRUE(llvm::isa<SaveNode>(oldSave));
```

Comment:
I think you can get rid of all this and skip to the second user of `A` directly.